docs/source/EccFramework.rst

```diff
@@ -0,0 +1,201 @@
+ECC Framework: Automatic Implementation and Evaluation of Error-Correcting Codes for Quantum Computing
+======================================================================================================
+
+The QECC library offers means for automatic implementation and evaluation of error-correcting codes for quantum computing. More precisely, the library allows to automatically apply different error correction schemes to quantum circuits provided as openqasm files. The "protected" quantum circuits can then be exported again in the form of openqasm files or can be directly used for noise-aware quantum circuit simulation. For the latter case, we also provide a wrapper script which makes use of the provided framework to apply error correction schemes to circuits and directly simulate those circuits using qiskit.
+
+Installation
+############
+
+Installation for Users
+----------
+
+.. code-block:: console
+
+  (venv) $ pip install mqt.qecc
+
+
+Installation for Developers
+----------
+
+If you have not done so already, clone the repository using:
+
+.. code-block:: console
+
+  git clone --recurse-submodules https://github.com/cda-tum/qecc
+
+Make sure you are in the main project directory for the next steps.
+
+.. code-block:: console
+
+  cd qecc
+
+and (if necessary), update the submodules.
+
+.. code-block:: console
+
+  git submodule update --init --recursive
+
+Then, the ECC framework can be installed using pip
+
+.. code-block:: console
+
+  (venv) pip install --editable .
+
+If you want to use Qiskit for quantum circuit simulation, you need to install it as well
+
+.. code-block:: console
+
+  (venv) pip install qiskit
+
+Usage
+#####
+
+Having the Python module installed, error correcting codes can be applied using apply_ecc of module qecc, like so
```

Comment:
Reword this a little. The spin should be more or less "to apply an Ecc to a quantum circuit use the :func:`~mqt.qecc.apply_ecc` function

Reply:
I changed the text to: 'Having the Python module installed, error correction can be applied to a quantum circuit using :func:~mqt.qecc.apply_ecc, like so'

Reply:
Nitpick: what is being installed is not a module, it's a package.
You then use a function that's defined in one of the package's modules.

Reply:
I switched the word to "package".